### PR TITLE
"and and" and other minor nits

### PR DESCRIPTION
**Pull request by kazuho**
In #1828, @martinthomson says:
> I saw a missing Oxford comma somewhere but I couldn't find it again when I opened the editor.

I could not find missing Oxford comma, but found some others that look odd to me (as a non-native).

### CODE REVIEW

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -523,12 +523,12 @@ HTTP/2 PRIORITY_UPDATE Frame {
 {: #fig-h2-reprioritization-frame title="HTTP/2 PRIORITY_UPDATE Frame Payload"}
 
 The Length, Type, Unused Flag(s), Reserved, and Stream Identifier fields are
-described in {{Section 4 of HTTP2}}. The frame payload of PRIORITY_UPDATE
-frame payload contains the following additional fields:
+described in {{Section 4 of HTTP2}}. The frame payload of PRIORITY_UPDATE frame
+contains the following additional fields:
```
Comment (LPardue) [inline draft-ietf-httpbis-priority.md:527]: or

```suggestion
described in {{Section 4 of HTTP2}}. The PRIORITY_UPDATE frame payload
contains the following additional fields:
```

File: draft-ietf-httpbis-priority.md [hunk 2]
```diff
@@ -523,12 +523,12 @@ HTTP/2 PRIORITY_UPDATE Frame {
...
 
 Reserved:
-: A reserved 1-bit field. The semantics of this bit are undefined, and the bit
-  MUST remain unset (0x0) when sending and MUST be ignored when receiving.
+: A reserved 1-bit field. The semantics of this bit is undefined. It MUST remain
+  unset (0x0) when sending and MUST be ignored when receiving.
```
Comment (LPardue) [inline draft-ietf-httpbis-priority.md:532]: the current text is verbatim how H2 defines it in https://httpwg.org/http2-spec/draft-ietf-httpbis-http2bis.html#section-4.1-4.8.1

The first part should remain "semantics ... are" due to grammar. The second part of the change is ok by me

**Review by LPardue — APPROVED** [review]
LGTM modulo one comment on the reserved bit

File: draft-ietf-httpbis-priority.md (hunk 2 above)
Comment (kazuho) [reply draft-ietf-httpbis-priority.md:532]: New Oxford American Dictionary says that the word "semantics" is "usually treated as singular," but I'm happy to not be American.

File: draft-ietf-httpbis-priority.md [hunk 3]
```diff
@@ -177,10 +177,11 @@ requiring less interpretation and allowing less variation.
...
 
-RFC 7540 priority is expressed relative to other requests on the same connection.
-Many requests are generated without knowledge of how other requests might share a
-connection, which makes this difficult to use reliably, especially in protocols
-that do not have strong ordering guarantees, like HTTP/3 {{HTTP3}}.
+RFC 7540 priority is expressed relative to other requests sharing the same
+connection at the same time. It is difficult to incorporate such design into
+applications that generate requests without knowledge of how other requests
+might share a connection, or into protocols that do not have strong ordering
+guarantees, like HTTP/3 {{HTTP3}}.
```
Comment (LPardue) [inline draft-ietf-httpbis-priority.md:184]: maybe "ordering guarantees across streams" or "ordering guarantees between requests" would be even clearer

File: draft-ietf-httpbis-priority.md (hunk 2 above)
Comment (kazuho) [reply draft-ietf-httpbis-priority.md]: Fixed in d3d71ba.

File: draft-ietf-httpbis-priority.md (hunk 3 above)
Comment (kazuho) [reply draft-ietf-httpbis-priority.md]: Good point! Went with "across streams," as H2 does not have ordering guarantee between _requests_.

**Pull request merged**